bin/pwa.dart

```diff
@@ -103,24 +103,24 @@ Future _generateManifestJson(String webDir, Map pubspec) async {
   await _updateIfNeeded(indexHtmlFile.path, indexHtmlContent);
 }
 
-/// If build/web is empty, run `pub build`.
+/// If build is empty, run `pub build`.
 Future _buildProjectIfEmptyOrOld(List<String> sources, List<String> excludes,
     List<String> offlineDirs) async {
   // This works only with the default value.
-  if (offlineDirs.length == 1 && offlineDirs.first == 'build/web') {
-    Directory dir = new Directory('build/web');
+  if (offlineDirs.length == 1 && offlineDirs.first == 'build') {
+    Directory dir = new Directory('build');
     if (dir.existsSync() && dir.listSync().isNotEmpty) {
       int lastSourceTimestamp = _getLastTimestamp(sources, excludes);
       int lastOfflineTimestamp = _getLastTimestamp(offlineDirs, null);
       if (lastSourceTimestamp < lastOfflineTimestamp) return;
     }
-    print('Running pub build:');
-    String executable = 'pub';
+    print('Running webdev build:');
+    String executable = 'webdev';
     if (Platform.isWindows) {
       try {
-        final pr = await Process.run('pub.exe', ['--version']);
+        final pr = await Process.run('webdev', ['--version']);
```

Comment:
Were you able to check this on Windows?

Reply:
I checked the behaviour on Windows, but neither pub nor webdev worked properly. I will test it on another machine to make sure that it is not something related to local installations or the like.

Reply:
This is working on Windows, since the user had built the app before. I will open an issue addressing that.